environment.yml

```diff
@@ -0,0 +1,9 @@
+  
```

Comment:
is this file needed by binder? How is it different than `postBuild`?

Reply:
This was required in one of the earlier versions. But I just figured that this was no longer required since the required packages are either installed using `postBuild` or `apt.txt` . Thanks for pointing this out! I'll remove this.